.github/workflows/e2e.js.yml

```diff
@@ -1,6 +1,6 @@
 # This workflow will run headful and headless e2e tests.
 
-name: e2e tests
+name: E2E tests
```

Comment:
These adjustments are just to make the badge names consistent in the `README.md`